src/Interface/Clients/DiscordIO.ts

```diff
@@ -7,8 +8,13 @@ import { ClientFiller } from '../ClientFiller'
  */
 export default class DiscordIO extends ClientFiller {
   get userCount() {
-    if (!this.client.users) return undefined
-    return Object.keys(this.client.users).length
+    if (!this.client.servers) return undefined
+    return Object.keys(this.client.servers)
+      .map((id) => this.client.servers[id])
```

Comment:
I think we can just use `Object.values`

```suggestion
    return Object.values(this.client.servers)
```